payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayRepository.kt

```diff
@@ -8,17 +8,16 @@ import com.google.android.gms.wallet.Wallet
 import com.stripe.android.GooglePayJsonFactory
 import com.stripe.android.core.Logger
 import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.filterNotNull
-import kotlinx.coroutines.flow.flowOf
 import javax.inject.Inject
 import javax.inject.Singleton
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
 
 fun interface GooglePayRepository {
-    fun isReady(): Flow<Boolean>
+    suspend fun isReady(): Boolean
```

Comment:
This is technically a breaking change, which would require a major release. We should add this to our backlog of [major release changes](https://docs.google.com/document/d/1xhjffAou-JvfVcdqF4He8J_prRQ_pw5Fqz3gWjEsLQc/edit)